include/time.h

```diff
@@ -240,6 +240,8 @@ int nanosleep(FAR const struct timespec *rqtp, FAR struct timespec *rmtp);
 void tzset(void);
 #endif
 
+#include <sys/time.h>
+
```

Comment:
This seems suspicious.  These are separate files with separate requirements and definitions.  Merging this does not seem to be in the spirit of the definitions of OpenGroup.org